rq/job.py

```diff
@@ -170,7 +183,7 @@ def create(
         on_failure: Optional[Union['Callback', Callable[..., Any]]] = None,  # Callable is deprecated
         on_stopped: Optional[Union['Callback', Callable[..., Any]]] = None,  # Callable is deprecated
```

Comment:
I can't comment on the first line, but you can change this to:

```suggestion
        on_failure: Optional[Union['Callback', FailureCallbackFunc]] = None,  # Callable is deprecated
        on_stopped: Optional[Union['Callback', StopCallbackFunc]] = None,  # Callable is deprecated
```

and you may want to make `Callback` generic so you can declare that as:

```suggestion
        on_failure: Optional[Union['Callback[FailureCallbackFunc]', FailureCallbackFunc]] = None,  # Callable is deprecated
        on_stopped: Optional[Union['Callback[StopCallbackFunc]', StopCallbackFunc]] = None,  # Callable is deprecated
```